music21/noteworthy/binaryTranslate.py

```diff
@@ -309,7 +317,7 @@ def advanceToNotNUL(self, nul=b'\x00'):
             pp += 1
         self.parsePosition = pp
 
-    def parse(self):
+    def parseContent(self):
```

Comment:
No strong arguments here. As far as I can remember it was a personnal readability helper, as several `parse` method are defined in this file (in different classes of course). Also, I thought it was more some kind of internal API than something people use from the other modules, but I can be wrong.

 I will revert if you prefer.

Reply:
> I thought it was more some kind of internal API than something people use from the other modules, but I can be wrong.

You are probably right that no one is using these helper methods, but since they're not marked as private, I just think it's safer to not break compatibility if there's no upside beyond code readability. See below for a post where I suggested that someone override helper methods the MIDI parser rather than rewrite the whole converter:

https://groups.google.com/g/music21list/c/ByloLCLCtLA/m/Vy9kNw7hBgAJ

Reply:
Oh, miss your message before pushing. I'm going to revert that and push again

Reply:
updated